qbft/types.go

```diff
@@ -27,6 +27,10 @@ type Syncer interface {
 	SyncDecidedByRange(identifier types.MessageID, from, to Height)
 }
 
+type RootGetter interface {
```

Comment:
this is only used in `qbft/spectest/comparable/types.go`, hence currently I don't see a point why it should be declared here. let's move it to `qbft/spectest/comparable/types.go` for now.

In general the best practice in go is to ACCEPT interfaces and declare them where they are accepted and not return Interfaces if you can and not declare them next to they're concrete types. (this is more of a java style like doing IRootGetter for any class)

Reply:
moved to types package because `ssv` package has types that implements the interface.
this also was mentioned before in this comment
https://github.com/bloxapp/ssv-spec/pull/191#discussion_r1172328133